org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/cleanup/StringConcatToTextBlockCleanUp.java

```diff
@@ -43,7 +43,7 @@ public ICleanUpFixCore createFix(CleanUpContextCore context) throws CoreExceptio
 		if (unit == null) {
 			return null;
 		}
-		return StringConcatToTextBlockFixCore.createCleanUp(unit);
+		return StringConcatToTextBlockFixCore.createCleanUp(unit, false);
```

Comment:
@JessicaJHee I'm not sure if we can get any option from `CleanUpConstants.STRINGCONCAT_STRINGBUFFER_STRINGBUILDER` and just set it to false to keep the original behavior. Could you check this?

Reply:
I think we can preserve the old behaviour for now, and create a separate cleanup / quickassist for the conversion. It should be fairly simple to do since it will just go through the same class (but with `true`).